src/pages/AddPage/AddPage.jsx

```diff
@@ -1,7 +1,44 @@
 import React from "react";
+import "./AddPage.scss";
+import { useState } from "react";
+import { apiBaseUrl } from "../../../config";
+import { requestDefaultHeaders } from "../../../config";
```

Comment:
Add apoBaseUr here. Do it in one import.

Reply:
Done.